src/cli/options.ts

```diff
@@ -168,6 +168,11 @@ export const optionMap = {
     alias: '--max-chunk-concurrency <maxChunkConcurrency>',
     description: 'Maximum number of concurrent chunks to upload per file',
   },
+  maxFinalizationWaitTimeMs: {
+    alias: '--max-finalization-wait-time-ms <maxFinalizationWaitTimeMs>',
```

Comment:
Nit: `time` in this already long param name is redundant. Could shorten to `--max-finalize-ms`